### DISCUSSION TITLE
Is there a way to customize advanced and general users

### DISCUSSION THREAD

**Discussion by wupengfei0904** [danny-avila/LibreChat#3419] [Q&A]
I'd like to assign different permissions to different users for gpt-4o and gpt-4o-mini access, so that I can differentiate between users who are willing to use the larger model and general users.

**Comment by taowang1993** [comment]
admin dashboard is on the roadmap but not sure if this feature will be included

**Reply by wupengfei0904** [reply]
How long will it take for the admin panel to go live?

**Reply by taowang1993** [reply]
not sure. if this feature is very important for you, you can check out anothe ui called open-webui. also a good one.
